packages/core/package.json

```diff
@@ -24,17 +24,19 @@
     "registry": "https://registry.npmjs.org"
   },
   "scripts": {
-    "build": "rm -rf ./build && yarn tsc"
+    "build": "rm -rf ./build && yarn tsc",
+    "unit-tests": "jest \".*unit.spec.ts\"",
+    "integration-tests": "jest --runInBand \".*integration.spec.ts\""
   },
   "dependencies": {
-    "@cosmjs/amino": "^0.28.4",
-    "@cosmjs/crypto": "^0.28.4",
-    "@cosmjs/encoding": "^0.28.4",
-    "@cosmjs/math": "^0.28.4",
-    "@cosmjs/proto-signing": "^0.28.4",
-    "@cosmjs/stargate": "^0.28.4",
-    "@cosmjs/tendermint-rpc": "^0.28.4",
-    "@cosmjs/utils": "^0.28.4",
+    "@cosmjs/amino": "^0.28.11",
+    "@cosmjs/crypto": "^0.28.11",
+    "@cosmjs/encoding": "^0.28.11",
+    "@cosmjs/math": "^0.28.11",
+    "@cosmjs/proto-signing": "^0.28.11",
+    "@cosmjs/stargate": "^0.28.11",
+    "@cosmjs/tendermint-rpc": "^0.28.11",
+    "@cosmjs/utils": "^0.28.11",
```

Comment:
I thought that using these versions of the libraries might have some impact considering they use `BigInt`. Should we upgrade them now?

Reply:
I think the `BigInt` problem was already present in the previous version as we used the` 0.28.4` version before.
If we wan't to fix this I think is beter to open a dedicated issue to downgrade the cosmjs lib back to 0.26.x

Reply:
Ok, perfect. I though it was only inside this new patch and not even older version. No need to open a new issue then